app/managers.py

```diff
@@ -0,0 +1,38 @@
+import sqlite3
+
+from app.models import Actor
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self.connection = sqlite3.connect("../cinema_db.sqlite")
+        self.table_name = "actors"
+
+    def create(self, first_name: str, last_name: str) -> None:
+        self.connection.execute(
+            f"INSERT INTO {self.table_name} "
+            f"(first_name, last_name) VALUES (?, ?)",
+            (first_name, last_name)
+        )
+
+    def all(self) -> list[Actor]:
+        actors_cursor = self.connection.execute(
+            f"SELECT * FROM {self.table_name}"
+        )
+        return [
+            Actor(id=actor_id, first_name=first_name, last_name=last_name)

```

Comment:
Simlify this to 
`return [Actor(*row) for row in actors_cursor]
`as it was in your previous commit